tuf_conformance/repository_simulator.py

```diff
@@ -21,8 +21,7 @@
     sim.snapshot.version += 1
 
     # As an exception, new root versions require explicit publishing
-    sim.root.version += 1
-    sim.publish_root()
+    sim.publish("root", bump_version=True)
```

Comment:
The example needs a bit of love :
* bump_version is not needed in this publish call 
* the example shows how root and snapshot changes are done differently: that's not true or useful anymore -- could remove one of those changes maybe?
*  `publish([Targets.type])` should be used instead of `sim.targets.version += 1`